config/config.go

```diff
@@ -156,10 +156,10 @@ type validatable interface {
 	Validate() error
 }
 
-// Unmarshallable defines an optional interface for custom configuration unmarshaling.
-// A configuration struct can implement this interface to override the default unmarshaling.
+// Unmarshallable defines an optional interface for custom configuration unmarshalling.
+// A configuration struct can implement this interface to override the default unmarshalling.
 type Unmarshallable interface {
-	// Unmarshal is a function that unmarshals a config.Map into the unmarshable struct in a custom way.
+	// Unmarshal is a function that unmarshalls a config.Map into the struct in a custom way.
```

Comment:
Shouldn't it still be `unmarshals`?
```suggestion
	// Unmarshal is a function that unmarshals a config.Map into the struct in a custom way.
```

Reply:
Which country :)))

Reply:
Don't know, I just asked Google and used the first hit as a justification to put a +1 there. :-)